oml/registry/datasets.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+from typing import Any, Dict, Tuple
+
+from oml.const import TCfg
+from oml.datasets.images import get_retrieval_images_datasets
+from oml.interfaces.datasets import ILabeledDataset, IQueryGalleryLabeledDataset
+from oml.registry.transforms import get_transforms_by_cfg
+
+
+def _build_image_dataset(**cfg: TCfg) -> Tuple[ILabeledDataset, IQueryGalleryLabeledDataset]:
```

Comment:
let's make naming similar to other modules
`get_image_datasets_by_cfg`
and
`get_image_datasets`

Reply:
Done